tests/commands/client_test.py

```diff
@@ -120,12 +120,12 @@ def test_http_get(self):
     def test_action_runs(self):
         self.client.action_runs('/api/jobs/name/0/act', num_lines=40)
         self.client.request.assert_called_with(
-            '/api/jobs/name/0/act?include_stdout=1&num_lines=40&include_stderr=1')
+            '/api/jobs/name/0/act?num_lines=40&include_stdout=1&include_stderr=1')
```

Comment:
Here's a diff that I believe accomplishes this (I also fixed the 5 space indent, wat)

```diff
diff --git a/tron/commands/client.py b/tron/commands/client.py
index 483bc2c..3bc2397 100644
--- a/tron/commands/client.py
+++ b/tron/commands/client.py
@@ -72,7 +72,7 @@ def request(uri, data=None):
 
 
 def build_get_url(url, data=None):
-     return '%s?%s' % (url, urllib.urlencode(data)) if data else url
+    return '%s?%s' % (url, urllib.urlencode(sorted(data.items()))) if data else url
 
 
 class Client(object):
```